modules/Connection.py

```diff
@@ -38,7 +38,24 @@ def get_request(target, key, session_id):
             print("Problem with get return of", target, ":", response.text)
             return False
 
-    # going to be used until reuse of session_id is in place
+    def post_request(target, key, post, session_id):
```

Comment:
I'd rename `post` to `data`

Reply:
But isn't it a post? Named at similar to the `get_request`

Reply:
I mean in the arguments. Otherwise it reads a little odd imho: `foo.post_request(..,.., post,..)`

Reply:
Ah, gotcha